src/borough/borough.repository.ts

```diff
@@ -59,6 +59,7 @@ export class BoroughRepository {
           boroughId: true,
         },
         where: eq(communityDistrict.boroughId, id),
+        orderBy: sql`${communityDistrict.id} ASC`,
```

Comment:
gee whiz: because we're not doing anything fancy with coercion, we should be able to use the drizzle functions

```suggestion
        orderBy: asc(communityDistrict.id),
```